packages/mobile/src/store/offline-downloads/slice.ts

```diff
@@ -23,10 +26,16 @@ export enum OfflineTrackDownloadStatus {
   ERROR = 'ERROR'
 }
 
+type CollectionDownloadPayload = {
+  collectionId: string
+  isFavoritesDownload: boolean
```

Comment:
Prob worth a comment here explaining what `isFavoritesDownload` means / what the two different types of downloaded collections are (i.e. from favoriting the collection and having "Download All Favorites" on vs. from directly downloading the collection)